core/templates/pages/contributor-dashboard-admin-page/contributor-admin-dashboard-page.component.spec.ts

```diff
@@ -422,19 +414,51 @@ describe('Contributor dashboard Admin page', () => {
       expect(component.selectedLanguage.id).toBe(nonDefaultLanguage.id);
     }));
 
-    it('should select last activity from dropdown', fakeAsync(() => {
+    it(
+      'should initially filters users by whether their last activity occurred ' +
+        'within the last 90 days',
+      fakeAsync(() => {
+        component.ngOnInit();
+        tick();
+        fixture.detectChanges();
+
+        expect(component.filter.maxDaysSinceLastActivity).toEqual(90);
+      })
+    );
+
+    it("should changes filter by users' last activity when end date changes", fakeAsync(() => {
```

Comment:
This and previous tests could be merged, no need to make them that small

Reply:
done